app/src/main/AndroidManifest.xml

```diff
@@ -47,17 +52,15 @@
             android:name=".view.activity.MainActivity"
             android:hardwareAccelerated="true"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar">
-        </activity>
+            android:theme="@style/AppTheme.NoActionBar"></activity>
 
-        <meta-data
-            android:name="io.fabric.ApiKey"
-            android:value="${fabric_api_key}"/>
+        <!-- <meta-data -->
```

Comment:
Not sure this should be commented

Reply:
If the project is still linked to frabric it shouldn't